src/library_nodefs.js

```diff
@@ -1,5 +1,5 @@
 mergeInto(LibraryManager.library, {
-  $NODEFS__deps: ['$FS', '$PATH'],
+  $NODEFS__deps: ['$PATH'],
```

Comment:
Oh wow, added update to my TODO list and it's been a while. I've removed $FS dependency from other files and added a comment as suggested above.